test/hack/cleanup/main.go

```diff
@@ -71,10 +76,32 @@ func main() {
 
 	// Terminate any old stacks that were provisioned as part of testing
 	// We execute these in serial since we will most likely get rate limited if we try to delete these too aggressively
+	sgInAccount := lo.Must(ec2Client.DescribeSecurityGroups(ctx, &ec2.DescribeSecurityGroupsInput{
```

Comment:
Can we just delete these security groups if they are beyond the expiration time like the other things?

Reply:
What about the VPC? Can we validate the age of the owning VPC and see if this is beyond the timestamp?

Reply:
The VPC does not have creation timestamp as well